tests/devices/test_default_mixed_autograd.py

```diff
@@ -115,6 +115,20 @@ def circuit():
 
         assert np.allclose(state, expected, atol=tol, rtol=0)
 
+    def test_no_ops(self):
```

Comment:
I wonder if these should be tested in `test_qnode.py` instead, as the relevant code is located in QNode. This problem will also apply to more than one device, as I bet it will occur with `DefaultQubit2` as well.

Reply:
Or perhaps `tests/interfaces/test_*_qnode.py`?